<!doctype html>
<html lang="en" prefix="og: http://ogp.me/ns#">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- SEO -->
    <title>Alfred Presentation</title>
    <meta name="description" content="Al.">

    <!-- Google Fonts -->
    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,700,700i%7CMaitree:200,300,400,600,700&amp;subset=latin-ext" rel="stylesheet">

    <!-- CSS Base -->
    <link rel="stylesheet" type='text/css' media='all' href="static/css/webslides.css">

    <!-- Optional - CSS SVG Icons (Font Awesome) -->
    <link rel="stylesheet" type="text/css" media="all" href="static/css/svg-icons.css">
  </head>
  <body>
    <main role="main">
      <article id="webslides" class="vertical">

        <!--title slide -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="aligncenter fadeInUp">
            <h1> <strong>Shotgun metagenomics </strong> </h1>
            <p> </p>
            <h3><strong> Alfred Ssekagiri </strong></h3>
            <p> </p>
            <h3><strong>Uganda Virus Research Institute - H3ABiONET</strong></h3>
              <p> </p>
          </div>
          <footer class="bg-trans-dark">
            <div class="wrap">
              <p>
                <span class="alignleft">
                  <figure>
                    <img img class="alignleft" src="pages/images/uvri2.png" alt="Tripal logo"> 
                  </figure>
                </span>
                <span class="alignright">
                  <figure>
                    <img img class="alignright" src="pages/images/bionet.jpg" alt="bionet logo"> 
                  </figure>
                </span>
              </p>
            </div>
          </footer>
        </section>

        <!--outline -->
        <section class="bg-apple" align="center">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1><strong>Outline</strong></h1>
                <p class="text-intro"> </p>
                <ul class="description">
                  <li>
                    <p class="text-intro"> <strong>Introduction to shotgun metagenomics</strong> </p>
                  </li>
                  <li>
                   <p class="text-intro"> <strong>Sample collection and extraction</strong> </p>
                  </li>
                  <li>
                   <p class="text-intro"> <strong>Library preparation and sequencing</strong></p>
                  </li>
                  <li>
                   <p class="text-intro"> <strong>Metagenome assembly</strong></p>
                  </li>
                  <li>
                   <p class="text-intro"> <strong>Metagenome annotation</strong></p>
                  </li>
                  <li>
                   <p class="text-intro"> <strong>Post analysis</strong></p>
                  </li>
                  </li>
                  <li>
                   <p class="text-intro"> <strong>Challenges and oppurtunities</strong></p>
                  </li>
                </ul>
          </div>

        </section>

        <!--Introduction -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1><strong>Introduction</strong></h1>
            <p class="text-intro"> </p>
                <ul class="description">
                  <li>
                    <p class="text-intro"><strong> What is it?: </strong> <br/> 
                      shotgun metagenomics, is the untargeted (‘shotgun’) sequencing of all (‘meta-’) microbial genomes ‘genomics’ present in a sample.
                    </p>
                  </li>
                  <li>
                    <p class="text-intro"><strong> What can it do?: </strong> <br/>
                     Shotgun sequencing can be used to profile taxonomic composition and functional potential of microbial communities and to recover whole genome sequences
                     </p>
                  </li>
                  <li>
                    <p class="text-intro"> <strong> Applications:  </strong>  <br/>
                      Understanding the functions and characterizing specific strains of these communities offers biotechnological promise in therapeutic discovery and innovative ways to synthesize products using microbial factories and can pinpoint the contributions of microorganisms to planetary, animal and human health.  </p>
                  </li>
                </ul>
          </div>
        </section>

        <!--work flow -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1> <strong>Shotgun metagenomics workflow </strong></h1>
              <figure>
                  <img img class="aligncenter" src="pages/images/workflow.png" alt="workflow"> 
                </figure>
          </div>
        </section>

        <!--Sample collection and dna extraction -  -->

        <!--library preparation and sequencing -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1><strong>Library preparation</strong></h1>
            <br/>
            <br/>
            <p class="text-intro"></p>        
                    <p class="text-intro"> 
                      <strong>Nextera DNA Flex Library Prep Kit:<br/>
                      </strong>
A fast, integrated workflow for a wide range of applications, from human whole-genome sequencing to amplicons, plasmids, and microbial species.
<br/><br/>
<strong>Nextera XT Library Prep Kit:<br/>
</strong>
Prepare sequencing-ready libraries for small genomes (bacteria, archaea, viruses), amplicons, and plasmids in less than 90 minutes.
<br/><br/>
<strong>TruSeq DNA PCR-Free Library Preparation Kits:<br/>
</strong>
Simple, all-inclusive library preparation for whole-genome sequencing applications. Researchers can sequence a wide variety of organisms, from small genomes such as bacteria to whole-human genomes.
                    </p>
          </div>
        </section>
        
        <!--sequencing -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1><strong>Sequencing platforms</strong></h1>
            <br/>
                    <p class="text-intro">
                     <strong>Requirement:<br/>
                     </strong> System that maximizes output in order to retrieve sequences from as many low-abundance members of the microbiome as possible. 
                    <br/><br/>
                      <strong>Examples:<br/>
                      </strong> 
                      Illumina HiSeq 2500 or 4000, 
                      NextSeq and NovaSeq produce high volumes of sequence data (between 120 Gb and 1.5 Tb per run).
                    <br/> <br/>
                    <strong>Preference:<br/>
                    </strong>
                    The Illumina HiSeq 2500, although now two generations old, is a popular choice for shotgun metagenomics, as it is able to generate 2 × 250-nt reads in rapid-run mode (generating up to 180 Gb per flow cell) or up to 1 Tb in high-output mode, with 2 × 125-nt reads.
                    </p> 
          </div>
        </section>

        <!--assembly -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1> <strong>Metagenome assembly </strong></h1>
              <br/>
              <p class="text-intro"> This is the process of recovering genomes from metagenomic data.</p>
              <p class="text-intro"><strong>Challenge(s)</strong>
              <br/>The coverage of each constituent genome depends on the abundance of each genome in the community. Low-abundance genomes may end up fragmented if overall sequencing depth is insufficient to form connections in the graph.<br/>
              A second problem is that a sample can contain different strains of the same bacterial species. These closely related genomes can cause branches in the assembly graph where they may differ by a single nucleotide variant or by the presence or absence of an entire gene or operon. The assembler will often stop at these branch points, resulting in fragmented reconstructions.</p>
              <p class="text-intro"><strong>Approach and resources</strong><br/>
              Most de novo metagenome assemblers are based on de Bruijn graphs: For single genome assemblies, a de Bruijn graph is constructed by breaking each sequencing read into overlapping subsequences of a fixed length k. This is different in for metagenomes due to above challenges. However, the assemblers are designed to overcome these challenges. </p>
              <p class="text-intro"><strong>Tools for metagenome assembly</strong><br/>
              metaVelvet<br/>
              metaSpades<br/>
              IDBA-UD<br/>
              Megahit</p>
          </div>
        </section>

        <!--Binning contigs -  -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1> <strong>Binning contigs </strong></h1>
              <br/>
              <p class="text-intro"><strong></strong>
              <br/> <strong>Aim: </strong>To group contigs into species. This is done through supervised  and unsupervised binning methods.<br/><br/>
              <strong>The princliple: </strong>
              The underlying principle is that contigs from the same genome will have similar coverage values within each metagenome.<br><br/>
              <strong>Approach: </strong>For taxonomic classification, contig homology against known genomes is a potentially useful approach, but most microbial species have not been sequenced, so a large fraction of reconstructed genomic fragments cannot be mapped to reference genomes.
              <br><br/>
              <!-- Contig coverage across multiple samples provides a much more powerful signal to group contigs together. <br><br/> -->
              <strong>Softwares</strong> automated for this include; METABAT, Groop, CONCOCT among others. 
              </p>
          </div>
        </section>

        <!--Assembly free metagenomic profiling-->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1> <strong>Assembly free metagenomic profiling</strong></h1>
              <br/>
              <p class="text-intro">
              <br/>
              Taxonomic profiling of metagenomes identifies which microbial species are present in a metagenome and estimates their abundance. <br/><br/>
              This can be carried out without assembly, through external sequence data resources, such as publicly available reference genomes.
              <br><br/>
              This approach can mitigate assembly problems, speed up computation and enable profiling of low-abundance organisms that cannot be assembled de novo. <br><br/>
              Main limitation is that previously uncharacterized microbes are difficult to profile
            </p>
          </div>
        </section>

        <!--Gene prediction and pathway analysis -  -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1> <strong>Genes and metabolic pathways</strong></h1>
              <br/>
              <p class="text-intro">
              <br/>
              With a fragmented but high-quality metagenome assembly, the genetic repertoire of a microbial community can be identified using adapted single-genome characterization tools.
              <br/><br/>
              Databases that include combinations of manually annotated and computationally predicted proteins families, such as KEGG or UniProt, can be used for this task and enable characterization of the functional potential of the microbiome
              <br><br/>
              A complementary approach to metabolic function profiling of metagenomes is an in-depth characterization of specific functions of interest. For example, identification of genes involved in antibiotic resistance (the ‘resistome’) in a microbial community can inform on the spread of antibiotic resistance.
            </p>
          </div>
        </section>

        <!--Assembly free profilling contigs -  -->
        <section class="bg-apple">
          <span class="background dark" style="background-image:url('pages/images/back.jpg')"></span>
          <div class="wrap">
            <h1> <strong>Post analysis</strong></h1>
              <br/>
              <p class="text-intro"><strong></strong>
              <br/>
              The outputs of the above mentioned processes comprise data matrices of samples versus microbial features (i.e., species, taxa, genes and pathways). Post- processing analysis uses statistical tools to interpret these matrices and establish how the findings correlate with the sample metadata.
              <br/>

              <br/>
              Post-processing tools include traditional multivariate statistics and machine learning. Unsupervised methods include simple clus- tering and correlation of samples, and visualization techniques such as heat maps, ordination, networks, which allow patterns in the data to be revealed graphically.
              <br/>
              <br/>
              Supervised methods include statistical methods, such as multivariate analysis of variance (ANOVA) for direct hypothesis testing of differences between groups, or machine learning classifiers that train models to label groups of samples, such as random forests
              <br/>
              <br/>
              A number of packages in R: DESeq2, vegan, metagenomeSeq among others.
            </p>
          </div>
        </section>

        <section class="bg-apple">
           <div>
           <figure>
                  <img img class="aligncenter" src="pages/images/post.png" alt="workflow"> 
                </figure>   
          </div>
        </section>

        <section class="bg-apple">
           <div>
           <figure>
                  <img img class="aligncenter" src="pages/images/thank.png" width='40%' alt="workflow"> 
                </figure>   
          </div>
          <footer class="bg-trans-dark">
            <div class="wrap">
              <p>
                <span class="alignleft">
                Alfred Ssekagiri
                </span>
                <span class="alignright">
                  <a href="#" title="Github">
                    <svg class="fa-github">
                      <use xlink:href="#fa-github"></use>
                    </svg>
                    https://www.github.com/AlfredUg
                  </a>
                </span>
              </p>
            </div>
          </footer>
        </section>
      </article>
    </main>
    <!--main-->
    <!-- Required -->
    <script src="static/js/webslides.js"></script>
    <script>
      window.ws = new WebSlides();
    </script>
    <!-- OPTIONAL - svg-icons.js (fontastic.me - Font Awesome as svg icons) -->
    <script defer src="static/js/svg-icons.js"></script>
  </body>
</html>
